Library/Homebrew/test/cask/artifact/app_spec.rb

```diff
@@ -334,5 +334,27 @@
 
       expect(contents_path).to exist
     end
+
+    it "properly handles non-writable directories" do
+      install_phase
+
+      contents_path = target_path.join("Contents")
+
+      expect(app.target).to receive(:writable?).at_least(:once).and_return false
+      allow(command).to receive(:run!).and_call_original
+      expect(command).to receive(:run!)
+        .with("/bin/cp", args: ["-pR", source_path.join("Contents"), target_path],
+                         sudo: true)
+      expect(command).to receive(:run!).with(any_args)
```

Comment:
Try moving the `.with(any_args)` to the `allow` above.

Reply:
Done.